modules/miner/blockmanager_test.go

```diff
@@ -0,0 +1,105 @@
+package miner
+
+import (
+	"errors"
+	"testing"
+
+	"github.com/NebulousLabs/Sia/types"
+)
+
+// reconstructBlock reconstructs a block given the miner and the header
+func reconstructBlock(m *Miner, header types.BlockHeader) (block *types.Block, err error) {
+	block, exists := m.blockMem[header]
+	if !exists {
+		err = errors.New("Header is either invalid or too old")
+		return
```

Comment:
Does Sia not use Go's named return values? (e.g. func foo() (a, b int) { ... } )